### PR TITLE
Final MVP

### PR DESCRIPTION
**Pull request by jsnorek**
This pull request is for the final MVP. It focuses on CSS and other visual additions like button icons.
This project's CSS is based around a small screen/mobile view size with larger screen media queries. 
Screenshot 2024-10-30 at 2 26 01 PM

Screenshot 2024-10-30 at 2 27 04 PM

On this PR I am also planning on adding error/success messages for certain features like logging in, creating a new user, and submitting a new review form.

### CODE REVIEW

**Comment by jsnorek** [comment]
@marjanasarker @ismith can you please take a look at this and let me know your thoughts?

As a note, I tried to get the .back-button in my CSS to keep it "stuck" to the top right corner but I couldn't get it to work correctly. I believe sticky doesn't work with the "overflow" in my container element, but even with changing that I still couldn't get it to work!

**Comment by jsnorek** [comment]
@MBHunter88 Hi! Can you please take a look at this when you have a chance?

**Review by MBHunter88 — APPROVED** [review]
The CSS looks great! I also loved that you implemented those updates for accessibility. The addition of the error messages for login really enhances the UX as well.

File: client/src/components/GameReviewForm.jsx [hunk 1]
```diff
@@ -3,11 +3,21 @@
...
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import axios from "axios";
+import { Rating } from "primereact/rating";
 
 function GameReviewForm({ setGameReviewFormVisible, gameDetails, loggedInUser, setNewReviewSubmitted, baseURL }) {
     
+    const [reviewFormSubmitMessage, setReviewFormSubmitMessage] = useState("");
+    
+    // Timer for error message
```
Comment (marjanasarker) [inline client/src/components/GameReviewForm.jsx]: what is the timer doing here?

**Review by marjanasarker — APPROVED** [review]
Beautifully done!! Bravo!! just one comment on what the timer was doing with the error messages. Additionally, if you want to keep some commented out code, leave a one liner on why you are leaving it there.

**Comment by ismith** [comment]
This looks fab! There's some bits of old code (commented out) that could be removed, but it's very demo-able as is.

File: client/src/components/GameReviewForm.jsx (hunk 1 above)
Comment (jsnorek) [reply client/src/components/GameReviewForm.jsx]: This timer causes the error message to disappear on the screen after it's been displayed for 5 seconds

**Comment by jsnorek** [comment]
@ismith thank you! I appreciate you taking a look at it :) I'll go back through and see what can be cleaned up

**Pull request merged**